### DISCUSSION TITLE
Online Accounts cannot add Google

### DISCUSSION THREAD

**Discussion by GianniJT** [elementary/triage#647] [Q&A]
When I try to add my google and microsoft account  via  Online Accounts, I get the same error  messages saying that  the account was unable to be saved because I am offline.  Clearly I am online.  I have tried a solution for v.7 which seemed to have worked for some.  That solution was installing the gnome control center and adding the account from there. However, this did not work.  Has anyone been able to solve this?

**Comment by teamcons** [comment]
What online accounts ? I thought Google and Microsoft were not supported OOTB on eOS
Did you do that on Evolution ? Which one, the one via sudo apt install ?

**Reply by GianniJT** [reply]
When setting up Mail in  Elementary v8 it points to " Online Accounts" in order to set up Mail.  After entering the requisite information ( name, email, password, name for the account) it then tries to connect and gives the error message referenced above.  I have tried this using the Mail app as a starting point and by going to System Settings and choosing the Online Accounts option under network .  Both  ask for the same info and generate the above referenced error.  In researching the support site I came upon an entry for someone using v.7 that had the same issue.  There was an answer that I tried to utilize that required installing gnome control center and entering the information from there.  Unfortunately, though I followed the instructions and was able to open the Gnome CC, I still could not add either my Google or my Micorsoft account.

**Comment by nandalo** [comment]
I used to do this:
https://www.google.com/amp/s/www.reallinuxuser.com/how-to-add-more-online-accounts-to-elementary-os-by-installing-gnome-control-center/amp/
But i dont try it on eos 8 yet...
